### PR TITLE
Fix QML Warnings

### PR DESCRIPTION
**Pull request by nephros**

### CODE REVIEW

File: src/qml/WebCatalogPage.qml [hunk 1]
```diff
@@ -170,11 +170,7 @@ Page {
 
             onClicked: {
                 pageStack.push(Qt.resolvedUrl("WebPatchPage.qml"),
-                               {
-                                   modelData: model,
-                                   delegate: background,
-                                   versions: versions
-                               })
+                               { modelData: model, versions: versions })
```
Comment (b100dian) [inline src/qml/WebCatalogPage.qml:173]: so delegate was not used here?

File: src/qml/patchmanager.h [hunk 2]
```diff
@@ -68,7 +68,7 @@ class PatchManager: public QObject
     Q_PROPERTY(bool applyOnBoot READ applyOnBoot WRITE setApplyOnBoot NOTIFY applyOnBootChanged)
     Q_PROPERTY(bool notifyOnSuccess READ notifyOnSuccess WRITE setNotifyOnSuccess NOTIFY notifyOnSuccessChanged)
     Q_PROPERTY(bool bitnessMangle READ bitnessMangle WRITE setBitnessMangle NOTIFY bitnessMangleChanged)
-    Q_PROPERTY(QStringList mangleCandidates READ mangleCandidates)
+    Q_PROPERTY(QStringList mangleCandidates READ mangleCandidates NOTIFY mangleCandidatesChanged)
```
Comment (b100dian) [inline src/qml/patchmanager.h:71]: Is this used now?

**Comment by nephros** [comment]
`[W] unknown:0 - QQmlExpression: Expression file:///usr/lib64/qt5/qml/org/SfietKonstantin/patchmanager/SettingsPage.qml:125:23 depends on non-NOTIFYable properties:
[W] unknown:42 - file:///usr/lib64/qt5/qml/org/SfietKonstantin/patchmanager/WebPatchPage.qml:42:1: QML WebPatchPage: <Unknown File>:10: Cannot assign to non-existent property "delegate"
[W] unknown:108 - file:///usr/lib64/qt5/qml/org/SfietKonstantin/patchmanager/WebPatchPage.qml:108:26: Unable to assign [undefined] to bool
[W] unknown:210 - file:///usr/lib64/qt5/qml/org/SfietKonstantin/patchmanager/WebPatchPage.qml:210:26: Unable to assign [undefined] to bool
[W] unknown:216 - file:///usr/lib64/qt5/qml/org/SfietKonstantin/patchmanager/WebPatchPage.qml:216:26: Unable to assign [undefined] to bool
[W] unknown:237 - file:///usr/lib64/qt5/qml/org/SfietKonstantin/patchmanager/WebPatchPage.qml:237:26: Unable to assign [undefined] to bool
[W] unknown:258 - file:///usr/lib64/qt5/qml/org/SfietKonstantin/patchmanager/WebPatchPage.qml:258:26: Unable to assign [undefined] to bool
[W] unknown:278 - file:///usr/lib64/qt5/qml/org/SfietKonstantin/patchmanager/WebPatchPage.qml:278:26: Unable to assign [undefined] to bool
[W] unknown:287 - file:///usr/lib64/qt5/qml/org/SfietKonstantin/patchmanager/WebPatchPage.qml:287:26: Unable to assign [undefined] to bool
[W] unknown:324 - file:///usr/lib64/qt5/qml/org/SfietKonstantin/patchmanager/WebPatchPage.qml:324:26: Unable to assign [undefined] to bool
[W] unknown:42 - file:///usr/lib64/qt5/qml/org/SfietKonstantin/patchmanager/WebPatchPage.qml:42:1: QML WebPatchPage: <Unknown File>:10: Cannot assign to non-existent property "delegate"
[W] unknown:108 - file:///usr/lib64/qt5/qml/org/SfietKonstantin/patchmanager/WebPatchPage.qml:108:26: Unable to assign [undefined] to bool
[W] unknown:210 - file:///usr/lib64/qt5/qml/org/SfietKonstantin/patchmanager/WebPatchPage.qml:210:26: Unable to assign [undefined] to bool
[W] unknown:216 - file:///usr/lib64/qt5/qml/org/SfietKonstantin/patchmanager/WebPatchPage.qml:216:26: Unable to assign [undefined] to bool
[W] unknown:237 - file:///usr/lib64/qt5/qml/org/SfietKonstantin/patchmanager/WebPatchPage.qml:237:26: Unable to assign [undefined] to bool
[W] unknown:258 - file:///usr/lib64/qt5/qml/org/SfietKonstantin/patchmanager/WebPatchPage.qml:258:26: Unable to assign [undefined] to bool
[W] unknown:278 - file:///usr/lib64/qt5/qml/org/SfietKonstantin/patchmanager/WebPatchPage.qml:278:26: Unable to assign [undefined] to bool
[W] unknown:287 - file:///usr/lib64/qt5/qml/org/SfietKonstantin/patchmanager/WebPatchPage.qml:287:26: Unable to assign [undefined] to bool
[W] unknown:324 - file:///usr/lib64/qt5/qml/org/SfietKonstantin/patchmanager/WebPatchPage.qml:324:26: Unable to assign [undefined] to bool
[W] unknown:42 - file:///usr/lib64/qt5/qml/org/SfietKonstantin/patchmanager/WebPatchPage.qml:42:1: QML WebPatchPage: <Unknown File>:10: Cannot assign to non-existent property "delegate"
[W] unknown:108 - file:///usr/lib64/qt5/qml/org/SfietKonstantin/patchmanager/WebPatchPage.qml:108:26: Unable to assign [undefined] to bool
[W] unknown:210 - file:///usr/lib64/qt5/qml/org/SfietKonstantin/patchmanager/WebPatchPage.qml:210:26: Unable to assign [undefined] to bool
[W] unknown:216 - file:///usr/lib64/qt5/qml/org/SfietKonstantin/patchmanager/WebPatchPage.qml:216:26: Unable to assign [undefined] to bool
[W] unknown:237 - file:///usr/lib64/qt5/qml/org/SfietKonstantin/patchmanager/WebPatchPage.qml:237:26: Unable to assign [undefined] to bool
[W] unknown:258 - file:///usr/lib64/qt5/qml/org/SfietKonstantin/patchmanager/WebPatchPage.qml:258:26: Unable to assign [undefined] to bool
[W] unknown:278 - file:///usr/lib64/qt5/qml/org/SfietKonstantin/patchmanager/WebPatchPage.qml:278:26: Unable to assign [undefined] to bool
[W] unknown:287 - file:///usr/lib64/qt5/qml/org/SfietKonstantin/patchmanager/WebPatchPage.qml:287:26: Unable to assign [undefined] to bool
[W] unknown:324 - file:///usr/lib64/qt5/qml/org/SfietKonstantin/patchmanager/WebPatchPage.qml:324:26: Unable to assign [undefined] to bool`

File: src/qml/WebCatalogPage.qml (hunk 1 above)
Comment (nephros) [reply src/qml/WebCatalogPage.qml:173]: No, it is/was used in other *PatchPage pages (which are now UnifiedPatchPage). WebPatchPage does not have or use the property, and spat a warning message. (See other comment for the log.)

File: src/qml/patchmanager.h (hunk 2 above)
Comment (nephros) [reply src/qml/patchmanager.h:71]: It was "used" in that it is part of an expression, which led to:

`[W] unknown:0 - QQmlExpression: Expression file:///usr/lib64/qt5/qml/org/SfietKonstantin/patchmanager/SettingsPage.qml:125:23 depends on non-NOTIFYable properties:`

**Review by b100dian — APPROVED** [review]
Thanks for the explanation & for fixing these!

**Pull request merged**